hyukju/app.js

```diff
@@ -41,6 +41,134 @@ app.post("/users", async (req, res, next) => {
   res.status(201).json({ message: "successfully created" });
 });
 
+app.post("/posts", async (req, res, next) => {
+  const { title, content, user_id } = req.body;
+  await DBDataSource.query(
```

Comment:
```suggestion
  await dbDataSource.query(
```
마찬가지로 자바스크립트에서는 클래스이름 혹은 enum과 같은 곳에는 대문자로 naming을 짓지만, 클래스로 만들어지는 인스턴스 또는 일반적인 경우에는 `dbDataSource`와 같이 소문자로 시작하는 네이밍을 활용합니다.